packages/package_image_1_4_1/tool_dependencies.xml

```diff
@@ -0,0 +1,17 @@
+<tool_dependency>
+    <package name="Image" version="1.4.1">
```

Comment:
Image is probably not a very good package name, but well, if the package is called image there is not mych you can do ...